lang/pl/index.md

```diff
@@ -244,22 +310,49 @@ w semantycznym wersjonowaniu chodzi przede wszystkim o przekazanie znaczenia
 zmiany poprzez zmianę numeru wersji. Jeśli zmiany są ważne dla użytkowników,
 poinformuj ich o tym poprzez numer wersji.
 
-### Jak powinienem radzić sobie z dezaprobowaniem funkcjonalności?
+### Jak powinienem radzić sobie z wycofywaniem funkcjonalności?
 
-Dezaprobowanie istniejącej funkcjonalności jest normalną częścią programowania
+Wycofywanie istniejącej funkcjonalności jest normalną częścią programowania
 i często jest konieczne, by móc rozwijać oprogramowanie. Gdy wycofujesz część
 swojego publicznego API, powinieneś zrobić dwie rzeczy: (1) zaktualizować
 dokumentację, by użytkownicy wiedzieli o tej zmianie, (2) wypuścić nowe wydanie
-minor z informacją o zdezaprobowaniu. Zanim całkowicie usuniesz funkcjonalność
+minor z informacją o wycofaniu. Zanim całkowicie usuniesz funkcjonalność
```

Comment:
```suggestion
minor z informacją o wycofywaniu. Zanim całkowicie usuniesz funkcjonalność
```
If we decide to change, I'd suggest rather "wycofywanie" here. I'd use not-finite form of the noun, as this is a process of withdrawing a functionality, not a one-off action.

Reply:
Yes. Sounds better.